style.css

```diff
@@ -0,0 +1,187 @@
+/* Media Queries*/
+@media only screen(max-width:500px) {
+    #sidebar{
```

Comment:
# css styling 
![Alt Text](https://media.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif)

- There is a missing space before the curly brace in Ln3
- I suggest you include a space before the curly brace for readability